react/Checkbox/Checkbox.less

```diff
@@ -1,5 +1,14 @@
 @import (reference) "~seek-style-guide/theme";
 
+.activeFocusReset() {
+  .input:not([readonly]):not([disabled]):focus + .label &,
+  .input:not([readonly]):not([disabled]):active + .label & {
```

Comment:
New version of styling correctly handles this indentation. Older one wanted 4 spaces here